ConcernsCaseWork/ConcernsCaseWork/Security/RbacManager.cs

```diff
@@ -27,9 +28,11 @@ public RbacManager(IConfiguration configuration, IUserRoleCachedService userRole
 			_defaultUsers = configuration["app:username"].Split(',');
 		}
 
-		public IList<string> GetDefaultUsers()
-		{
-			return UserRoleMap.GetDefaultUsersExcludeE2E(_defaultUsers);
+		public Task<IList<string>> GetSystemUsers(params string[] excludes)
+		{			
+			return Task.FromResult(
```

Comment:
Is there a reason why not to use await here?

Reply:
OK I guess the data is hardcoded so nothing really to await. Fine with this.

Reply:
yep